apps/recovery-utility/renderer/components/RecoveryForm/index.tsx

```diff
@@ -174,15 +175,22 @@ export const RecoveryForm = ({ verifyOnly }: Props) => {
                 {...register('agpRsaPassphrase')}
               />
             ) : (
-              <TextField
-                id='passphrase'
-                type='password'
-                label='Mobile App Recovery Passphrase'
-                helpText='Set by the workspace owner during onboarding'
-                error={errors.passphrase?.message}
-                disabled={recoverMutation.isLoading}
-                {...register('passphrase')}
-              />
+              <Tooltip title={recoverOnlyNCW ? 'Mobile passphrase is not required for NCW wallets recovery' : ''}>
+                <TextField
+                  id='passphrase'
+                  type={recoverOnlyNCW ? 'text' : 'password'}
```

Comment:
No need for this, you can keep it like it was before

Reply:
Are you referring to the tooltip?

Reply:
Just the last part of type

Reply:
Without this change, the button to unhide the PW is still visible when the field is disabled and it looks bad...

Reply:
lgtm